pyvista/core/utilities/fileio.py

```diff
@@ -1073,33 +1072,24 @@ def to_meshio(mesh: DataSet) -> meshio.Mesh:
     connectivity = mesh.cell_connectivity
 
     # Generate polyhedral cell faces if any
-    if pyvista.vtk_version_info >= (9, 4):
-        polyhedral_cells = mesh.polyhedron_faces
-        if polyhedral_cells is None:
-            msg = 'We have not yet implemented polyhedral cells for VTK 9.4+'
-            raise VTKVersionError(msg)
-        locations = mesh.polyhedron_face_locations
-    else:
-        polyhedral_cells = mesh.faces
-        locations = mesh.face_locations
+    def split(arr: VectorLike[int]) -> VectorLike[VectorLike[int]]:
```

Comment:
If there is already a function I am not aware of that does that, please let me know.